src/Training.AirBnb.Clone.Backend/AirBnB.Infrastructure/AirBnB.Infrastructure.csproj

```diff
@@ -6,6 +6,10 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="FluentValidation.DependencyInjectionExtensions" Version="11.8.1" />
```

Comment:
Why we need dependency injection here ? Can't we just use `FluentValidation` package ?